src/components/FormCharacteristics.js

```diff
@@ -112,7 +112,9 @@ export const FormCharacteristics = ({ charId, setCount }) => {
           <input className='char-value' type='number' min='0' max='5' name='willpower' value={willpower} onChange={onChange} />
         </label>
       </div>
+
       {validated === false && <p>Characteristics must have a value between 1 and 5</p>}
```

Comment:
Noticed that Characteristics error message read "must be between 1 and 5" but Skills error message reads "must be between 0 and 5." 

Are they different on purpose?